src/include/storage/irc_catalog.hpp

```diff
@@ -28,7 +28,7 @@ class ICRClearCacheFunction : public TableFunction {
 	static void ClearCacheOnSetting(ClientContext &context, SetScope scope, Value &parameter);
 };
 
-enum class ICEBERG_CATALOG_TYPE { AWS_S3TABLES, AWS_GLUE, OTHER, INVALID };
+enum class ICEBERG_CATALOG_TYPE { AWS_S3TABLES, AWS_GLUE, POLARIS, INVALID };
```

Comment:
Any reason this needs to specifically be Polaris? `AWS_S3Tables` and `AWS_GLUE` exist because there is some special handling when building the endpoint, and also to trigger the logic to make a SigV4 request instead of a regular CURL request. 

I think for catalogs that support Oauth, `OTHER` should be fine. (For instance Lakekeeper also uses Oauth)

Reply:
This change is not necessary for this PR

Reply:
Reverted